src/Workleap.DomainEventPropagation.Subscription.PullDelivery.Tests/CloudEventHandlerUnitTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System.Reflection;
+using Azure.Messaging;
+using FluentAssertions;
```

Comment:
Nothing wrong with FluentAssertions, mais pour un concern d'uniformité on peut ne pas l'utiliser svp? On s'était déjà poser la question et on avait décidé de pas utiliser pour raison de préférence 🙂

Reply:
Good to know, this should have been taken care of with #114